Expensify/App
# fix: Emoji Picker Styles

### Details
1. Set the Fixed width on Emoji Picker for Web & 100% for Mobile web.

### Fixed Issues

Fixes #2446

### Tests \ QA Steps

   1. Log in to expensify cash
   2.  Go to a conversation and click on the emoji icon
   3. Search for an emoji.
   4. Emoji picker box's size should not change.


### Tested On


### Screenshots


#### Web


https://user-images.githubusercontent.com/24370807/118410438-100fcb00-b6ad-11eb-9fd0-f99818351245.mp4



#### Mobile Web


#### Desktop


#### iOS


#### Android
